package.json

```diff
@@ -68,6 +68,9 @@
     ]
   },
   "jest": {
+    "setupFiles": [
+      "<rootDir>/test/__mocks__/requestAnimationFrame.js"
```

Comment:
@shiawuen Why was it necessary to add this?

Reply:
@davidtheclark Because of this https://facebook.github.io/react/blog/#javascript-environment-requirements

Reply:
React also depends on requestAnimationFrame (even in test environments).

Reply:
Aha, thanks!